NugetLightRequest.cs

```diff
@@ -1587,7 +1612,12 @@ into newGroup
                     }
                     else
                     {
-                        pkgs = from p in pkgs where p.IsLatestVersion select p;
+                        var latestPkg = pkgs.First();
+                        foreach (var p in pkgs) {
+                            if (p.Version > latestPkg.Version) {
+                                latestPkg = p;
+                            }
+                        }
```

Comment:
With this an unlisted latest version gets selected even when other versions are still listed. 
For example: 
A depends on B, 
B's versions are 1.0, 2.0.3.0
B's 3.0 version got unlisted
When you install package 'A', we should not install the 3.0 (unlisted) version of B.